### DISCUSSION TITLE
Passing headless into chrome options doesn't work and gives errors

### DISCUSSION THREAD

**Discussion by rawalvarun** [aahnik/wappdriver#40] [General]
Passing headless into chrome options doesn't work and gives errors

**Comment by aahnik** [comment]
It will not work, because i have not written code in that way. I will add this feature in future. 

For headless to work, we need we extract the qr code and save it to a file, and refresh that rapidly.

Some other changes may also need to be made.

Headless mode will be supported in future. It is in my [target](https://github.com/aahnik/wappdriver/projects/1#card-44998349).
